docs/perf/usage/index.md

```diff
@@ -86,6 +86,22 @@ async function customTrace() {
 }
 ```
 
+## Custom screen traces
+
+Record a custom screen rendering trace (slow frames / frozen frames)
+
+```jsx
+import perf from '@react-native-firebase/perf';
+
+async function screenTrace() {
+  // Define & start a screen trace
+  const trace = await perf().startScrenTrace('FooScreen');
```

Comment:
make the example do a try/catch style with mention that iOS rejects, and Android 8/8.1 rejects, as discussed in other comments

Reply:
fixed